test/cluster/pg-snapshot-resumption/03-while-storaged-down.td

```diff
@@ -7,5 +7,9 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
+> SET transaction_isolation = serializable
```

Comment:
Do you want to add a comment here to why this is necessary?

Reply:
:+1:

Reply:
Just added one.